sql/sql_acl.cc

```diff
@@ -14393,6 +14398,41 @@ static bool check_password_lifetime(THD *thd, const ACL_USER &acl_user)
   return false;
 }
 
+static void handle_nonexistent_user_login_failed(THD *thd,
```

Comment:
@vuvova What do you think about this ?

Reply:
@vuvova following up on this, it looks like it needs a reply from you.